Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -834,6 +835,7 @@
                         FontFamily="{StaticResource FluentUIGlyphs}"
                         Header="{x:Bind Header, Mode=OneWay}"
                         IconSource="{x:Bind IconSource, Mode=OneWay}"
+                        IsTabStop="False"
```

Comment:
Can you confirm that the first tab can still be focused using the tab key?

Reply:
No, this disables it. If the plan is still keep them on the tab cycle then this PR isn't useful. I wasn't thinking anyone would want to tab onto these. I'm closing the PR then, thanks for your time.